app/client/src/sagas/ActionExecution/PluginActionSagaUtils.ts

```
@@ -24,3 +30,30 @@ export function* setDefaultActionDisplayFormat(
     );
   }
 }
+
+export const createActionExecutionResponse = (
+  response: ActionExecutionResponse,
+): ActionResponse => {
+  const payload = response.data;
+
+  if (payload.statusCode === "200 OK" && payload.hasOwnProperty("headers")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)